cEP-0005.md

```diff
@@ -124,6 +124,17 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+Before moving any further, it is quite necessary to have a clear-cut definition of both aspects and tastes, to avoid any kind of discrepancies and chaos in the future.
```

Comment:
I'm for a more concrete approach, this whole sentence doesn't add any info, the title in line 129 could hold this info easily. `## Naming of aspects and tastes`